datastore.tf

```diff
@@ -65,5 +65,5 @@ module "lambda_datastore" {
   rds_tags      = var.rds_tags
   s3_tags       = var.s3_tags
   dynamodb_tags = var.dynamodb_tags
-  tags          = merge({ "Lambda Application" = var.application_name }, var.datastore_tags, var.tags)
+  tags          = merge({ "Lambda Application" = var.application_name }, var.datastore_tags, { "version" = var.application_version }, var.tags)
```

Comment:
This is interesting adding the version tag to other resources. I'm not sure I agree that the version of the service should exist on non Lambda function resources? Is the application version tag relevant?

Reply:
Application version is the version of lambda deployed which points to package.json version. Having this could be helpful in debugging?